### PR TITLE
nit edits to the website

### PR DESCRIPTION
**Pull request by rgrandl**

### CODE REVIEW

File: website/docs.md [hunk 1]
```diff
@@ -721,7 +727,7 @@ id. Then comes the file and line where the log was produced, followed finally by
 the contents of the log.
 
 The main component returned by `weaver.Init` has a Logger method as well (like
-all `weaver.Instance`s):
+all `weaver.Instances`):
```
Comment (spetrovic77) [inline website/docs.md:730]: I'd revert this as `weaver.Instance` is a thing.

File: website/docs.md [hunk 2]
```diff
@@ -746,7 +752,7 @@ fooLogger = logger.With("foo", "bar")
 fooLogger.Info("A log with attributes.")  // adds foo="bar"
 ```
 
-Note that you can also add normal print statements to your code. These prints
+**Note** that you can also add normal print statements to your code. These prints
```
Comment (spetrovic77) [inline website/docs.md:755]: Can you do the same format for these, which should be:

**Note**: You can also...

Same below.

File: website/docs.md [hunk 3]
```diff
@@ -1845,7 +1853,7 @@ to open a dashboard in a web browser.
 TODO(mwhittaker): Remove rollout section?
 </div>
 
-Note that `weaver gke` configures GKE to autoscale your application. As the load
+**Note** that `weaver gke` configures GKE to autoscale your application. As the load
```
Comment (spetrovic77) [inline website/docs.md:1856]: Ditto

**Review by spetrovic77 — APPROVED** [review]

File: website/docs.md (hunk 1 above)
Comment (rgrandl) [reply website/docs.md:730]: It looks ugly, hence I changed it. It displays as weaver.Instance s. It's not a big deal though.

File: website/docs.md (hunk 3 above)
Comment (rgrandl) [reply website/docs.md]: Done.

**Pull request merged**